### ISSUE TITLE
Gateway timeout when downloading in test_contrib_text.test_download_embed

### ISSUE THREAD

**Issue by KellenSunderland** [apache/incubator-mxnet#9669]
## Description

The test 'test_contrib_text.test_download_embed' is failing in CI.  Example: http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-9609/31/pipeline/432

The failure seems to be a gateway timeout when trying to download from the url https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/embeddings/test/embedding_test.vec

This url does seem reachable externally.

## Detailed Test Failure
```
======================================================================
ERROR: test_contrib_text.test_download_embed
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Anaconda3\envs\py2\lib\site-packages\nose\case.py", line 197, in runTest
    self.test(*self.arg)
  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\unittest\test_contrib_text.py", line 131, in test_download_embed
    test_embed = text.embedding.create('test')
  File "C:\jenkins_slave\workspace\ut-python-gpu\pkg_vc14_gpu\python\mxnet\contrib\text\embedding.py", line 86, in create
    return create_text_embedding(embedding_name, **kwargs)
  File "C:\jenkins_slave\workspace\ut-python-gpu\pkg_vc14_gpu\python\mxnet\registry.py", line 163, in create
    return registry[name](*args, **kwargs)
  File "C:\jenkins_slave\workspace\ut-python-gpu\tests\python\unittest\test_contrib_text.py", line 127, in __init__
    pretrained_file_path = Test._get_pretrained_file(embedding_root, pretrained_file_name)
  File "C:\jenkins_slave\workspace\ut-python-gpu\pkg_vc14_gpu\python\mxnet\contrib\text\embedding.py", line 220, in _get_pretrained_file
    download(url, downloaded_file_path, sha1_hash=expected_downloaded_hash)
  File "C:\jenkins_slave\workspace\ut-python-gpu\pkg_vc14_gpu\python\mxnet\gluon\utils.py", line 208, in download
    raise RuntimeError("Failed downloading url %s"%url)
RuntimeError: Failed downloading url https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/embeddings/test/embedding_test.vec
-------------------- >> begin captured stdout << ---------------------
Downloading embeddings\test\embedding_test.vec from https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com/gluon/embeddings/test/embedding_test.vec...

--------------------- >> end captured stdout << ----------------------
-------------------- >> begin captured logging << --------------------
requests.packages.urllib3.connectionpool: DEBUG: Starting new HTTPS connection (1): apache-mxnet.s3-accelerate.dualstack.amazonaws.com
requests.packages.urllib3.connectionpool: DEBUG: https://apache-mxnet.s3-accelerate.dualstack.amazonaws.com:443 "GET /gluon/embeddings/test/embedding_test.vec HTTP/1.1" 504 624
--------------------- >> end captured logging << ---------------------
```

## Environment info (Required)

Windows Python 2 env on CI.

@szha: Possible to label this one as flaky?

**Comment by ThomasDelteil** [comment]
@KellenSunderland this should be solved with https://github.com/apache/incubator-mxnet/pull/11181 please consider closing

**Comment by piyushghai** [comment]
@marcoabreu Shall we consider this problem as solved with #1181 ?

**Comment by marcoabreu** [comment]
Oh yeah, thanks a lot!
